@AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageTSDDraw.sqf

```diff
@@ -174,6 +174,87 @@ if (count _fcrTargets > 0) then {
     _pointsArray pushBack [MPD_POSMODE_WORLD, _pos, "", POINT_TYPE_FCR, _forEachIndex, _ident];
 } forEach _fcrTargets;
 
+//ASE Points
+
+private _aseObjects  = _heli getVariable "fza_ah64_ase_rlwrObjects";
+private _radius      = 1.00;
+{
+    _x params ["_state", "_bearing", "_classification"];
+    private _ident      = "";
+    private _unitStatus = "";
+    private _unitType   = "";
+    //Unit type
+    switch (_classification) do {
```

Comment:
Can we deduplicate the code between `pageASEDraw` and `pageTSDDraw` by adding a new function?

Reply:
Not sure, they're two unique implementations. The ASE page is fixed, whereas the TSD page has to account for centered vs. decentered.